sources/packages/backend/libs/integrations/src/sfas-integration/sims-sfas-files/sims-to-sfas.footer.ts

```diff
@@ -0,0 +1,30 @@
+import { SpecializedStringBuilder } from "@sims/utilities";
+import { SIMSToSFASBaseRecord } from "./sims-to-sfas-base.record";
+
+/**
+ * Footer record for SIMS to SFAS file.
+ */
+export class SIMSToSFASFooter extends SIMSToSFASBaseRecord {
+  /**
+   * Total record count of all records that are not header and trailer records.
+   ** There is no clear requirement currently to map this field. It is populated with filler values.
```

Comment:
I was just looking at the doc again and the example provided says to just do a total SUM of record types 200, 300, 301, and 400 (everything but header and footer).
I do not recall now if there is an open question about it or if it is the final decision.

Reply:
I saw the doc too. I am also not sure if what is there in the doc is a final decision. Because our conversations with @JasonCTang wasn't about the total count. We can check with business/Jason and I can add the count.